Changes

```diff
@@ -1,8 +1,27 @@
 Release history for Zonemaster component Zonemaster-CLI 
 
+v3.0.4 2021-02-10 (public release version)
+
+ [Features]
+ - Updates --raw mode output in debug level (#113)
+
+ [Fixes]
+ - Updates the installation instructions (#188)
+ - Prevents PO files to be updated unintentionally (#186)
```

Comment:
Could we have a separate heading for changes that aren't user facing?

Reply:
Yes, we could. I am more than willing to have a template for the Changes files. Do you want to have a change in this Changes, or could that be for next release? I have created an issue, https://github.com/zonemaster/zonemaster/issues/930